### ISSUE TITLE
Array shapes trigger FullyQualifiedClassNameInAnnotation sniff

### ISSUE THREAD

**Issue by szepeviktor** [slevomat/coding-standard#792]
```
 54 | ERROR | [x] Class name \Toolkit4WP\name in @return should be referenced via a fully qualified name.
    |       |     (SlevomatCodingStandard.Namespaces.FullyQualifiedClassNameInAnnotation.NonFullyQualifiedClassName)
 54 | ERROR | [x] Class name \Toolkit4WP\priority in @return should be referenced via a fully qualified name.
    |       |     (SlevomatCodingStandard.Namespaces.FullyQualifiedClassNameInAnnotation.NonFullyQualifiedClassName)
```

Here is the code

```php
    /**
     * Read hook tag from docblock.
     *
     * mindplay/annotations may be a better solution.
     *
     * Format: @hook hook_name 10
     *
     * @return array{name: string, priority: int}|null
     */
    protected function getMetadata(string $docComment, int $defaultPriority): ?array
    {
        $matches = [];
        if (
            \preg_match(
                '/^\s+\*\s+@hook\s+([\w\/-]+)(?:\s+(\d+))?\s*$/m',
                $docComment,
                $matches
            ) !== 1
        ) {
            return null;
        }

        return ['name' => $matches[1], 'priority' => \intval($matches[2] ?? $defaultPriority)];
    }
```

Please help.

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/b0137099f391c81b3bef12d30621f62c29a9a9d3

**Comment by szepeviktor** [comment]
Thank you.
